src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -22,6 +30,15 @@ private class FormatterState
             private static TaggedText s_newlinePart = new TaggedText(TextTags.LineBreak, "\r\n");
 
             internal readonly List<TaggedText> Builder = new List<TaggedText>();
+            private readonly List<(DocumentationCommentListType type, int index, bool renderedItem)> _listStack = new List<(DocumentationCommentListType type, int index, bool renderedItem)>();
+            private readonly Stack<(string target, string hint)> _navigationTargetStack = new Stack<(string target, string hint)>();
+            private readonly Stack<TaggedTextStyle> _styleStack = new Stack<TaggedTextStyle>();
```

Comment:
docs on this stuff would be helpful.  not sure why we need stacks and whatnot (for the nav or list bit).   Also, why is _listStack not an actual Stack?  

Basically, it's a bit confusing here, so docs would help :)

Reply:
I'm documenting all the items from this except for the following:

> Also, why is `_listStack` not an actual Stack?

Elements of the list are accessed by index.